src/Symfony/Component/HttpKernel/CacheWarmer/CacheWarmerAggregate.php

```diff
@@ -106,6 +108,11 @@ public function warmUp(string $cacheDir)
                 file_put_contents($this->deprecationLogsFilepath, serialize(array_values($collectedLogs)));
             }
         }
+
+        $preload = array_merge([], ...$preload);
+        $preload = array_keys(array_flip($preload));
```

Comment:
isn't that an `array_unique`? why write it like this?

Reply:
Because that's how I wrote it... now updated.